pkg/server/objects_operations.go

```diff
@@ -2,38 +2,78 @@ package server
 
 import (
 	"context"
+	"time"
 
 	"github.com/codenotary/immudb/pkg/api/schema"
 	"github.com/codenotary/immudb/pkg/api/schemav2"
 	"github.com/golang/protobuf/ptypes/empty"
 )
 
 func (s *ImmuServer) LoginV2(ctx context.Context, loginReq *schemav2.LoginRequest) (*schemav2.LoginResponseV2, error) {
-	return nil, nil
+	username := []byte(loginReq.Username)
+	password := []byte(loginReq.Password)
+	session, err := s.OpenSession(ctx, &schema.OpenSessionRequest{
+		Username:     username,
+		Password:     password,
+		DatabaseName: loginReq.Database,
+	})
+	if err != nil {
+		return nil, err
+	}
+	expirationTimestamp := int32(0)
+	if s.Options.SessionsOptions.MaxSessionInactivityTime > 0 {
+		expirationTimestamp = int32(time.Now().Add(s.Options.SessionsOptions.MaxSessionInactivityTime).Unix())
+	}
+	return &schemav2.LoginResponseV2{
+		Token:               session.SessionID,
```

Comment:
what about being explicit with the value included into the response? i.e. SesssionUUID or SessionID